web/groups.go

```diff
@@ -189,7 +169,7 @@ func (s *QuickFeedService) getGroupUsers(request *qf.Group) ([]*qf.User, error)
 
 	users, err := s.db.GetUsers(userIds...)
 	if err != nil {
-		return nil, err
+		return nil, connect.NewError(connect.CodeInternal, errors.New("failed to get users"))
```

Comment:
Applies for all cases: We should return just `err` from helper methods like this one so that the original error can be logged at the call site in `quickfeed_service.go` only, where we should do the `connect.NewError` creation.